client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -646,4 +682,74 @@ public BeanParamProperty(Function<Object, Object> getter, Annotation[] annotatio
             this.annotations = annotations;
         }
     }
+
+    /**
+     * Class used when the full response is needed
+     * @param <T> the type of data to be returned
+     */
+    public static class Response<T> {
```

Comment:
Is there any sane way of keeping these three classes out of HttpClient? It's already huge :)

Reply:
What a good suggestion. Had them here before when i tried to have them private. But not there is no need. I fix